src/include/ipc/topology.h

```diff
@@ -87,8 +87,9 @@ struct sof_ipc_comp {
 #define SOF_MEM_CAPS_CACHE		BIT(6) /**< cacheable */
 #define SOF_MEM_CAPS_EXEC		BIT(7) /**< executable */
 #define SOF_MEM_CAPS_L3			BIT(8) /**< L3 memory */
+#define SOF_MEM_CAPS_MMU_SHD		BIT(9) /**< MMU shared memory */
 /* Don't forget to update when adding a new bit to the ABI. */
-#define SOF_MEM_CAPS_LOWEST_INVALID	BIT(9) /**< Used for input validation */
+#define SOF_MEM_CAPS_LOWEST_INVALID	BIT(10) /**< Used for input validation */
```

Comment:
Lets not add caps, since this is user ABI, lets add flags instead (which is not ABI). Pls see #10088 as it adds a flag for SOF_MEM_FLAG_KERNEL and SOF_MEM_FLAG_USER.